core/src/mindustry/content/Bullets.java

```diff
@@ -17,29 +17,29 @@
     public static BulletType
 
     //artillery
-    artilleryDense, artilleryPlastic, artilleryPlasticFrag, artilleryHoming, artilleryIncendiary, artilleryExplosive,
+    artilleryDense, artilleryPlastic, artilleryPlasticFrag, artilleryHoming, artilleryIncendiary, artilleryExplosive, artilleryFortress,
```

Comment:
Why are you putting all of these bullets back here? I *intentionally* removed them from the bullet list. Barring low tiers, units should not reuse projectiles from other units.

Reply:
Reverted commit